document-service-api/src/api/openapi.yaml

```diff
@@ -0,0 +1,205 @@
+---
+openapi: 3.0.0
+info:
+  title: Appeals Document Service API Documentation
+  version: 1.0.0
+  license:
+    name: www.foundry4.com
+servers:
+  - url: http://localhost:3001/
+paths:
+  "/api/v1/{applicationId}":
```

Comment:
Perhaps the applicationId could be provided(required) in the header because each calling app will have their own fixed value.

Reply:
I don't get it?